devstack/settings

```diff
@@ -25,6 +25,16 @@ PROMETHEUS_CONTAINER_REPOSITORY=${PROMETHEUS_CONTAINER_REPOSITORY:-quay.io/prome
 PROMETHEUS_CONTAINER_TAG=${PROMETHEUS_CONTAINER_TAG:-latest}
 PROMETHEUS_CONTAINER_IMAGE=$PROMETHEUS_CONTAINER_REPOSITORY:$PROMETHEUS_CONTAINER_TAG
 
+### node_exporter ###
+
+# Note: We need to add <host_ip:9100> in PROMETHEUS_CUSTOM_SCRAPE_TARGETS.
+# It will allow node exporter to send data to prometheus server.
```

Comment:
Since you're adding code and variables for automatically deploying the node exporter, I think it'd make sense to add a scrapeconfig for it too https://github.com/openstack-k8s-operators/sg-core/tree/main/devstack/prometheus-files/scrape_configs . With that you'll be able to use the `PROMETHEUS_SERVICE_SCRAPE_TARGETS=node-exporter` instead of having to remember <host_ip:port>

Reply:
Done, thank you!

Reply:
looks good